docs/contributing.md

```diff
@@ -59,7 +59,12 @@ The docs are a collection of [Markdown files][markdown] that also utilize
 For more information on how to use GitDocs read the [GitDocs
 guide][gitdocs_guide].
 
-# Regarding style
+# Regarding language & style
+
+We ask that you avoid trivializing terms when contributing to documentation.
+This includes words like "just", "simply", "easy", "obvious", and
```

Comment:
I know this is annoying, but I think we usually keep punctuation like commas and periods inside of quotations. I'm sure there are differing opinions on this, but I usually just default to MLA:

`"just," "simply," "easy," "obvious," and "straightforward."`

Not a reason not to merge, but seems best to remain consistent! :)

Reply:
@jacobherrington looks like this got merged by @nickytonline before i could make this change! sorry about that. can probably come back to it at a later point, looks like we have other instances of this like in `architecture.md`

Reply:
Oops my bad..

Reply:
lol no biggie, it's not important :)